tests/tests/connectors/test-class-connector-edd.php

```diff
@@ -18,9 +18,6 @@ public function setUp() {
 
 		$this->plugin->connectors->unload_connectors();
 
-		$post_connector = new Connector_Posts();
```

Comment:
This was causing the `wp_stream_log_data` action to be called twice -- once for the EDD connector and once for the Posts connector and the test in this file was being run during both.